### ISSUE TITLE
Support for Code Folding

### ISSUE THREAD

**Issue by meoblast001** [YsuSERESL/iTrace#13]
Currently code folding is not supported in iTrace. If lines 2 through 24 are folded into one line, and the user looks at line 25, the recorded line number will be 3.

**Comment by shbonita** [comment]
@nickiovino https://eclipse.org/articles/Article-Folding-in-Eclipse-Text-Editors/folding.html

**Comment by shbonita** [comment]
This has been implemented in Issue# 46

**Comment by onebenclark** [comment]
Waiting on pull request.
